src/lib.rs

```diff
@@ -187,6 +187,25 @@ fn concat<T>(mut v1: Vec<T>, mut v2: Vec<T>) -> Vec<T> {
     v1
 }
 
+fn check_for_default(triples: &mut Vec<(TokenStream, Ident, TokenTree)>) {
+    let mut default_position: Option<usize> = None;
+    for (attributes, variant_name, variant_value) in triples.into_iter() {
```

Comment:
I get a warning about unused variable `variant_name`. Might want to rename the variable `_variant_name` or `_` to suppress the warning.